src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
+  var mainDiv = element.parentElement.parentElement;
+  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
+  var childDiv = element.nextElementSibling;
+  var menuID = childDiv.id;
+
+  for (i = 0; i < menuItems.length; i++) {
+    if (menuID !== menuItems[i].id) {
+      menuItems[i].className = mainDiv.className.includes(
+        "dg_allowmutlipleopen"
+      )
+        ? menuItems[i].className
+        : menuState("close");
+    } else {
+      if (menuItems[i].className.includes(menuOpen)) {
+        menuItems[i].className = menuState("close");
+        menuItems[i].setAttribute("aria-expanded", false);
+        element.parentElement.className = "card fa collapsed dg_menuitem";
+      } else {
+        menuItems[i].className = menuState("open");
+        menuItems[i].setAttribute("aria-expanded", true);
+        element.parentElement.className = "card fa dg_menuitem";
+      }
+    }
+  }
+}
+
+function AllMenuItemsAction(button) {
+  var status = button.innerHTML.trim();
+  var body = button.parentElement;
+  var menuItems = body.getElementsByClassName("multi-collapse");
+
+  if (button.innerHTML.trim() === buttonOpenAll) {
```

Comment:
this can be simplified to use a ternary

Reply:
I was under the impression we were dong refactoring afterwards not during the code reviews. Since that isnt the case I have made these modificiations.